### ISSUE TITLE
Support fetchOrderBooks for multiple symbols at once

### ISSUE THREAD

**Issue by pmizel** [ccxt/ccxt#1033]
Like in #932 was asked, 
there are some exchanges which support fetchOrderBook / Groups / Full / All
i try to create one pr.

- OS: win
- Programming Language: javascript
- CCXT version:
- Exchange: exmo, tidex, cryptopia, yobit, c-cx
- Method:

```
↑↑↑ DO_NOT_DELETE_THE_BACKTICKS_AROUND_THE_CODE

MUST_READ_THIS https://github.com/kroitor/ccxt/wiki/Manual#troubleshooting

REPLACE_WITH_YOUR_CODE_TO_REPRODUCE_THE_BUG_HERE

example: 

if(ccex.hasFetchOrderBookGroup)
ccex({'verbose': True}).fetchOrderBookGroup('ETH/BTC','ETH/BTC',...)

REPLACE_WITH_YOUR_VERBOSE_OUTPUT_HERE

↓↓↓ DO_NOT_DELETE_THE_BACKTICKS_AROUND_THE_CODE
```

**Comment by kroitor** [comment]
Hi! Thanks! It would be awesome to support those! However, I have one concern on keeping it readable, I'd suggest, let's not call it `hasFetchOrderBookGroup + fetchOrderBookGroup` and instead name it `hasFetchOrderBooks + fetchOrderBooks`. That would be consistent with other methods in the library. What do you think about it?

**Comment by pmizel** [comment]
thx, we keep it readable.

**Comment by kroitor** [comment]
@pmizel one more thing, I forgot to tell, please, only commit JS files in a pull request. Most other files are generated and your changes to .py files and tables in docs will be overwritten upon merge as described here: https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#how-to-contribute-code.

**Comment by kroitor** [comment]
Merged #1050 into version 1.10.1024! Thx!
